contents/docs/session-replay/privacy.mdx

```diff
@@ -94,4 +99,67 @@ To do so, you should add the CSS class name `ph-no-capture` to elements which sh
 <div class="ph-no-capture">I won't be captured at all!</div>
 ```
 
-Note that adding `ph-no-capture` will also prevent any [autocapture](/docs/data/autocapture) events from being captured from that element.
\ No newline at end of file
+Note that adding `ph-no-capture` will also prevent any [autocapture](/docs/data/autocapture) events from being captured from that element.
+
+
+# Common example configs
```

Comment:
For some reason this is not shown in the preview. https://posthog-36hnvs1z2-post-hog.vercel.app/docs/session-replay/privacy

Perhaps because it should be H2 and not H1?
```suggestion
## Common example configs
```